pkg/nas/nodeserver.go

```diff
@@ -314,6 +314,10 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 		return &csi.NodePublishVolumeResponse{}, nil
 	}
 
+	if strings.Contains(opt.Server, ":") {
```

Comment:
Maybe someone would specify an IPv6 address in the `Server` field?

Reply:
https://github.com/stefanha/nfs-utils/blob/708a963f5eb1892e39db1470d46429d8d8e17e38/utils/mount/parse_dev.c#L210

Here is how mount.nfs parse it. Maybe we can mirror its behavior.